modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16970,7 +16974,9 @@ sort-portlet-instance-configuration-name=Sort
 sort-structures-by-name=Sort Structures by Name
 sort-structures-by-name-help=Set this to true to show structures sorted by ascending name when adding a web content. Otherwise, structures will be sorted by identifier. Enabling this property can degrade performance depending on the number of existing ddmStructures since sorting is done in memory.
 sortable=Sortable
+sortable-column=Sortable column
 sorting=Sorting
+sorted-by-column-x-in-x-order=Sorted by column {0} in {1} order
```

Comment:
Actually, since it's not a full sentence so I wouldn't punctuate it. It looks okay to me as-is, but this is my first look at a  language key that's only for a screen reader.

Reply:
Coming back to this again https://github.com/brianchandotcom/liferay-portal/pull/151583#issuecomment-2180404211. Any idea?

Reply:
@rbohl what do you think?

Reply:
In my opinion you don't want to add words just to make it a sentence, since it's quite clear as is and will only be read by the screen reader I'd just make it title case, _Sorted by Column {0} in {1} Order_ 
@ethib137 @matuzalemsteles

Reply:
Sounds good. @matuzalemsteles can you apply this change and forward to BChan again? Thanks.